frontend/src/AlertsManagement/AlertInfo.js

```diff
@@ -46,14 +50,30 @@ class AlertInfo extends Component {
     newAlert.status = statusToChangeTo;
     this.setState({ alert: newAlert });
   }
+
+  handlePriorityChange = (newPriority) => {
+    const { alert } = this.state;
+
+    // We have to get the P0, P1, or P2 version to match with enum representation in backend.
+    const numToEnum = Object.keys(priorityLevels)[Object.values(priorityLevels).indexOf(newPriority)];
+    fetch('/api/v1/alert-visualization', {
+      method: 'POST',
+      body: alert.id + " " + numToEnum,
```

Comment:
oh i think this is why the post servlet above (in the previous file I commented had the space in the body), could we not change this?

Reply:
refer to earlier comment - I will change this to passing through parameters in the rquest rather than the body